components/layout_2020/flow/text_run.rs

```diff
@@ -76,7 +76,7 @@ enum SegmentStartSoftWrapPolicy {
 
 #[derive(Debug, Serialize)]
 pub(crate) struct TextRunSegment {
-    /// The index of this font in the parent [`InlineFormattingContext`]'s collection of font
+    /// The index of this font in the parent [`super::InlineFormattingContext`]'s collection of font
```

Comment:
This fix and the similar one below are still useful actually! Do you mind rebasing your change on top of the latest?

Reply:
let me do that right away

Reply:
@mrobinson, on rebasing, I do not see the changes of #31587. It seems they aren't merged to main yet